static/js/message_list_view.js

```diff
@@ -1134,6 +1134,13 @@ export class MessageListView {
         this._maybe_format_me_message(message_container);
         this._add_msg_edited_vars(message_container);
 
+        // The timestr of message_container can be outdated
+        // so, as the timestamp was updated in echo.js for that
+        // This updates the timestr and also the .message_time
+        // for the message that is being re rendered
+        set_timestr(message_container);
+        row.find(".message_time").text(message_container.timestr);
```

Comment:
Do we need both of these lines?  My read of the `row.replaceWith()` below would be that we should only need the `set_timestr` line here.

Reply:
Yes, I checked it right now. We don't need the second line. Only updating the `set_timestr` updates the timestamp.